src/ui/public/notify/notifier.js

```diff
@@ -97,10 +97,14 @@ function add(notif, cb) {
     });
   } else if (notif.customActions) {
     // wrap all of the custom functions in a close
-    notif.customActions = notif.customActions.map(action => {
+    notif.customActions = notif.customActions.map((action, idx) => {
```

Comment:
It's a built-in of the args to the callback from Array.prototype.map: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

```
callback
  Function that produces an element of the new Array, taking three arguments:

  currentValue (I called it `action`)
    - The current element being processed in the array.
  index (I called it `idx`)
    - The index of the current element being processed in the array.
  array (not using this one)
    - The array map was called upon.

```